### ISSUE TITLE
set the value at startup and cannot be changed later

### ISSUE THREAD

**Issue by DominikTesch** [Yakindu/statecharts#287]
reported via forum

I encountered the same issue. (I'm using the latest Yakindu bundle for Windows, 64-bit).

Please see the attached example. The simulator allows to set the value of event E_1 only once, at startup. Then, the event value is not reset and cannot be changed. The edge with guard [valueof(E_1)==42] can only be traversed at startup (if the correct value of the event is set), and never again.

Am I doing something wrong, or is it an issue of the tool?

![default](https://cloud.githubusercontent.com/assets/15800400/12198184/f11abd0e-b60d-11e5-85a3-cd79d1fd8ff9.png)
![default](https://cloud.githubusercontent.com/assets/15800400/12198187/f3b80ec2-b60d-11e5-8b31-9f76eb230ead.jpg)

**Comment by andreasmuelder** [comment]
This is a duplicate of #329

**Comment by DominikTesch** [comment]
The view does not update itself.

**Comment by arianaa30** [comment]
Is there an estimation of when it will be fixed?

**Comment by andreasmuelder** [comment]
the bug is already fixed, there is only a minor update problem is the view. This will be rolled out in SCT 2.6.0.

**Comment by andreasmuelder** [comment]
Fixed
